From d3013f26c4bb4c54f20e3ad90dee41cc5404a0ec Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Wed, 12 Jul 2023 10:30:35 +0200
Subject: [PATCH 323/954] feat(stm32mp2): fix macros for watchdog's OTP

The OTP for stm32mp2 watchdog is named HCONF1 / OTP124.
Rename the macros accordingly.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git d96e1d64b95a1a56c93afdd61d63ca8ca32be490 feat(stm32mp2): fix macros for watchdog's OTP]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I3dbdd8b0613dc93ea36b86045781462c5cccfbe6
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/317861
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 plat/st/stm32mp2/stm32mp2_def.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/plat/st/stm32mp2/stm32mp2_def.h b/plat/st/stm32mp2/stm32mp2_def.h
index f98c797252..8ca0ecbdf7 100644
--- a/plat/st/stm32mp2/stm32mp2_def.h
+++ b/plat/st/stm32mp2/stm32mp2_def.h
@@ -308,7 +308,7 @@ enum ddr_type {
 /* OTP labels */
 #define PART_NUMBER_OTP				"part_number_otp"
 #define PACKAGE_OTP				"package_otp"
-#define HW2_OTP					"hw2_otp"
+#define HCONF1_OTP				"otp124"
 #define NAND_OTP				"otp16"
 #define NAND2_OTP				"otp20"
 #define BOARD_ID_OTP				"board_id"
@@ -323,9 +323,9 @@ enum ddr_type {
 #define PACKAGE_OTP_PKG_SHIFT			0
 
 /* IWDG OTP */
-#define HW2_OTP_IWDG_HW_POS			U(0)
-#define HW2_OTP_IWDG_FZ_STOP_POS		U(1)
-#define HW2_OTP_IWDG_FZ_STANDBY_POS		U(2)
+#define HCONF1_OTP_IWDG_HW_POS			U(0)
+#define HCONF1_OTP_IWDG_FZ_STOP_POS		U(1)
+#define HCONF1_OTP_IWDG_FZ_STANDBY_POS		U(2)
 
 /* NAND OTP */
 /* NAND parameter storage flag */
-- 
2.53.0

